services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/core/application.py

```diff
@@ -50,8 +51,13 @@ def create_app(settings: ApplicationSettings) -> FastAPI:
     if settings.RESOURCE_USAGE_TRACKER_POSTGRES:
         setup_db(app)
     setup_redis(app)
+    setup_rabbitmq(app)
 
-    setup_background_task(app)
+    # NOTE: This task currently runs in the background and collects data from prometheus for all
+    # jupyter smash services. This is currently not needed.
+    # setup_background_task(app)
```

Comment:
just remove it. git is here for history.

Reply:
done :+1: